miasm2/expression/__init__.py

```diff
@@ -16,3 +16,4 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+"Intermediate language implementation"
```

Comment:
To highlight the difference with ir, you could say that this module is about abstract expression manipulation a